frontend/package.json

```diff
@@ -43,7 +43,7 @@
     "yup": "^1.2.0"
   },
   "scripts": {
-    "start": "PORT=3001 react-scripts start",
```

Comment:
Could we leave the frontend port as 3001?

Reply:
We can change the user service port to 3003 instead since matching service uses port 3002

Reply:
This is fixed!